iree/compiler/Conversion/LinalgToSPIRV/SplitDispatchFunctionPass.cpp

```diff
@@ -67,9 +68,8 @@ bool canSeparateOps(ArrayRef<Operation *> ops) {
   for (auto currOp = ops.begin(), nextOp = std::next(ops.begin());
        nextOp != ops.end(); ++currOp, ++nextOp) {
     Operation *iter = (*currOp)->getNextNode();
-    while (isa<linalg::ReshapeOp, SubViewOp>(iter)) {
+    while (iter != *nextOp && MemoryEffectOpInterface::hasNoEffect(iter))
```

Comment:
Add trivial braces back to follow IREE's coding style.

Is it corrected by clang-tidy? Maybe you need to add something like `-checks="readability-braces-around-statements"`.

Reply:
I think IREE follows LLVM style (in compiler/) except this one. One day I got a review comment and looked into other files, I found that we did follow this in other files. Ben told me that the omission of braces in the LLVM style has always really confused him, and sticking with the Google style here is much nicer.

I am a person who actually prefer to have braces if it is not in a single line. Because it looks clearer to me. Anyway, I just follow the style that more codes used in IREE codebase. :p